src/main/java/mate/hwdao/db/Storage.java

```diff
@@ -0,0 +1,14 @@
+package mate.hwdao.db;
+
+
+import java.util.ArrayList;
+import java.util.List;
+import mate.hwdao.model.Car;
+import mate.hwdao.model.Driver;
+import mate.hwdao.model.Manufacturer;
+
+public class Storage {
+    public static final List<Car> listCar = new ArrayList<>();
+    public static final List<Driver> listDriver = new ArrayList<>();
```

Comment:
it isn't a requirement, we need to have the only a list of Manufacturer

Reply:
\+
that's the code from next hws